staging/src/k8s.io/apiserver/pkg/admission/metrics/metrics.go

```diff
@@ -153,19 +153,31 @@ func (m *AdmissionMetrics) ObserveWebhook(elapsed time.Duration, rejected bool,
 }
 
 type metricSet struct {
-	latencies        *prometheus.HistogramVec
-	latenciesSummary *prometheus.SummaryVec
+	latencies                  *prometheus.HistogramVec
+	deprecatedLatencies        *prometheus.HistogramVec
+	latenciesSummary           *prometheus.SummaryVec
+	deprecatedLatenciesSummary *prometheus.SummaryVec
 }
 
 func newMetricSet(name string, labels []string, helpTemplate string, hasSummary bool) *metricSet {
-	var summary *prometheus.SummaryVec
+	var summary, deprecatedSummary *prometheus.SummaryVec
 	if hasSummary {
 		summary = prometheus.NewSummaryVec(
 			prometheus.SummaryOpts{
 				Namespace: namespace,
 				Subsystem: subsystem,
 				Name:      fmt.Sprintf("%s_admission_latencies_seconds_summary", name),
-				Help:      fmt.Sprintf(helpTemplate, "latency summary"),
+				Help:      fmt.Sprintf(helpTemplate, "latency summary in seconds"),
+				MaxAge:    latencySummaryMaxAge,
+			},
+			labels,
+		)
+		deprecatedSummary = prometheus.NewSummaryVec(
+			prometheus.SummaryOpts{
+				Namespace: namespace,
+				Subsystem: subsystem,
+				Name:      fmt.Sprintf("%s_admission_latencies_milliseconds_summary", name),
+				Help:      fmt.Sprintf(helpTemplate, "latency summary in milliseconds (deprecated)"),
```

Comment:
We should be consistent with the deprecation warning. Let’s make sure the help text is preceded with (Deprecated) like the other metrics we have already deprecated.

Reply:
@brancz fixed.